prov/verbs/src/fi_verbs.c

```diff
@@ -905,29 +844,57 @@ fi_ibv_msg_ep_sendmsg(struct fid_ep *ep, const struct fi_msg *msg, uint64_t flag
 }
 
 static ssize_t
-fi_ibv_msg_ep_recvmsg(struct fid_ep *ep, const struct fi_msg *msg, uint64_t flags)
+fi_ibv_msg_ep_send(struct fid_ep *ep, const void *buf, size_t len,
+		void *desc, fi_addr_t dest_addr, void *context)
 {
-	struct fi_ibv_msg_ep *_ep;
-	struct ibv_recv_wr wr, *bad;
-	struct ibv_sge *sge=NULL;
-	size_t i;
+	const struct iovec iov = {
+		.iov_base = (void *)buf,
```

Comment:
I added that since the compiler was warning about const to non const assignment. .iov_base is not const while buf is.